bin/auditwheel

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+
+# Patch to not ship CUDA system libraries
+# Follows https://github.com/DIPlib/diplib/tree/master/tools/travis
+import sys
+
+from auditwheel.main import main
+from auditwheel.policy import _POLICIES as POLICIES
+
+# Do not include licensed dynamic libraries
+libs = [
+    "libcudart.so.11.0",
+    "libcublasLt.so.11",
+    "libcublas.so.11",
+    "libcusparse.so.11",
+    "libcustatevec.so.1",
```

Comment:
I think this was asked before, but don't we want wildcards on the lib versions?

Reply:
As I recall, I wasn't able to validate this worked in the past. In the interest of time, and since we pin explicitly to these lib version (their names have remained invariant through symlinks) we should be fine with this until we move to CUDA 12.

Reply:
I'm curious about it too.